### DISCUSSION TITLE
Each minute: "node service is not as expected. setting node service to enabled"

### DISCUSSION THREAD

**Discussion by vinzent** [noobaa/noobaa-core#7826] [General]
We have 3 OpenShift 4.13 clusters with NooBaa (as part of ODF).

Each instance logs every minute:

> node service is not as expected. setting node service to enabled

The log message originates from here: 

https://github.com/noobaa/noobaa-core/blob/2edff0b16d8f600459b2a68bad6723976ac4d217/src/server/node_services/nodes_monitor.js#L1253-L1256

As nothing has changed, I would expect that the flow would end in return on line 1254 instead.

Why does NooBaa think something has changed each minute?

**Comment by liranmauda** [comment]
Hi @vinzent 
This seems to be a bug.
we are populating `location_info.region` if the `item_pool` exists, regardless if it is `undefined`

Then when we do `_.isEqual` it will fail.

Great catch,
Can you create an issue from this discussion, so we can handle it? 

Best Regards,
Liran.

**Comment by liranmauda** [comment]
In any case, https://github.com/noobaa/noobaa-core/pull/7831

**Reply by vinzent** [reply]
Thanks a lot for the quick-fix! Should I still open an issue?

**Reply by liranmauda** [reply]
yes, Please open an Issue 
I will close this discussion.
